### ISSUE TITLE
fullchain.pem

### ISSUE THREAD

**Issue by goodeats** [certbot/certbot#3403]
following along on this blog: http://collectiveidea.com/blog/archives/2016/01/12/lets-encrypt-with-a-rails-app-on-heroku/

I get to this step successfully:

IMPORTANT NOTES:
 - Congratulations! Your certificate and chain have been saved at
   /etc/letsencrypt/live/inchworm.io/fullchain.pem. Your cert will
   expire on 2016-04-11. To obtain a new version of the certificate in
   the future, simply run Let's Encrypt again.

When I look in the /live folder it's empty. What details could I provide to help solve this issue since I need to update the cert in heroku from here.

Thanks.

**Comment by cowlicks** [comment]
Did you re-deploy your app to heroku with the correct challenge before continuing through the certbot menu?

**Comment by goodeats** [comment]
Yes, http://mydomain.com/.well-known/acme-challenge/long-key-here returns the expected string on the page as I have it stored in my config variables.

**Comment by bmw** [comment]
Really? Does `/etc/letsencrypt` exist? Is there anything in `/etc/letsencrypt/archive`?

**Comment by goodeats** [comment]
`etc/letsencrypt` exists with the following subdirectories: `accounts`, `archive`, `csr`, `keys`, `live`, `renewal`

`csr` has a `0000_csr-certbot.pem` file
`renewal` has a `mydomain.com.conf` file
the rest are empty

I should add that I'm using homebrew to install

**Comment by goodeats** [comment]
Ok I've solved my issue.

I was not having permission to view these folders. I noticed this when I ran `sudo -H /bin/bash` to see that the files were indeed there. I came across this article which referenced issues with homebrew installations: https://whyistheinternetbroken.wordpress.com/2016/03/25/docker-mac-install-fails/

Once I reset my permissions I was good to go. I completed the steps and now I have SSL on my site!
